packages/jest-emotion/test/matchers.test.js

```diff
@@ -67,9 +67,11 @@ describe('toHaveStyleRule', () => {
       const wrapper = enzyme[method](<Component />)
       expect(wrapper).toHaveStyleRule('color', 'red')
       expect(wrapper).not.toHaveStyleRule('width', '100%')
-      const svgNode = wrapper.find('svg')
-      expect(svgNode).toHaveStyleRule('width', '100%')
-      expect(svgNode).not.toHaveStyleRule('color', 'red')
+      if (method !== 'shallow') {
```

Comment:
If we're using a shallow render, we won't be able to find the svg element, as it will be wrapped - this is expected behavior, and inline with other CSS-in-JS libs.

Reply:
could we maybe split that into a separate test then? Seems like pre condition and post condition assertions are testing slightly different things here